upstream.Dockerfile

```diff
@@ -14,6 +14,7 @@ WORKDIR /
 COPY --from=builder /go/src/github.com/operator-framework/operator-lifecycle-manager/bin/olm /bin/olm
 COPY --from=builder /go/src/github.com/operator-framework/operator-lifecycle-manager/bin/catalog /bin/catalog
 COPY --from=builder /go/src/github.com/operator-framework/operator-lifecycle-manager/bin/package-server /bin/package-server
+COPY --from=builder /go/src/github.com/operator-framework/operator-lifecycle-manager/bin/mock-ext-server /bin/mock-ext-server
```

Comment:
do we want this in non-test containers?

Reply:
Yes you are correct. This should only be in test containers.